util/argo/argo.go

```diff
@@ -75,6 +75,20 @@ func FilterByRepo(apps []argoappv1.Application, repo string) []argoappv1.Applica
 	return items
 }
 
+// FilterByCluster returns an application
+func FilterByCluster(apps []argoappv1.Application, cluster string) []argoappv1.Application {
+	if cluster == "" {
+		return apps
+	}
+	items := make([]argoappv1.Application, 0)
+	for i := 0; i < len(apps); i++ {
+		if apps[i].Spec.Destination.Server == cluster {
```

Comment:
Do we need to support filter apps by cluster names? What if the user provides a cluster name instead of url?

Reply:
Sure, I can make another PR to do the job

Reply:
I think you can add one more condition here to check cluster by its symbolic name
```
if apps[i].Spec.Destination.Server == cluster ||  apps[i].Spec.Destination.Name == cluster {
```

Reply:
Good idea!